llabake/Hello-Books
# Ft signup page using react 153751184

#### What does this PR do?
This PR implements the sign up page using react
#### Description of Task to be completed?
User should be able to enter form data to sign up to the aplication
#### How should this be manually tested?
- clone the repo
- cd into the project folder
- install all needed dependency using _**npm install**_
- visit the port where the webpack-dev-server started, the signup page should be loaded there
#### What are the relevant pivotal tracker stories?
153751184
#### Any background context you want to add?
N/A
#### Screenshots
